server/gitlab/gitlab.go

```diff
@@ -20,6 +20,7 @@ var (
 	ErrPrivateResource = errors.New("private resource")
 )
 
+//go:generate mockgen -source=gitlab.go -destination=mocks/mock_gitlab.go
```

Comment:
Should this command be in `mocks` instead?

Reply:
no this command simply help any other user easily update the mock after any code change in interface as well as the build and test can automatically create the mock files/structs.